roofit/roofitcore/inc/RooArgSet.h

```diff
@@ -21,12 +21,14 @@
 class RooArgList ;
 
 
-#define USEMEMPOOL
+#define USEMEMPOOLFORARGSET
```

Comment:
Is there a case where the guard is not set?

Reply:
I don't see what you mean. It's not a guard, but a switch. However, the switch (old name) was switching both memory pools for RooArgSet or RooDataSet, or only one - depending on the order of inclusion.
Therefore, I changed the names of both switches such that each only switches on/off for one of the two sets.